app/src/androidTest/java/com/example/android/architecture/blueprints/todoapp/test/chapter1/actions/RecyclerViewActionsTest.java

```diff
@@ -59,6 +64,13 @@ public void addNewToDosChained() throws Exception {
         onView(withId(R.id.tasks_list)).perform(scrollToPosition(2));
     }
 
+    @Test
+    public void refreshListOfTasksByUsingSwipe() throws  Exception {
+        generateToDos(2);
+        onView(withId(R.id.tasks_list))
+                .perform(customSwipeActions.swipeCustom(10, GeneralLocation.TOP_CENTER,GeneralLocation.CENTER));
```

Comment:
Don't forget a space between method parameters. You missed it here after first one of those two: `GeneralLocation.TOP_CENTER,GeneralLocation.CENTER`

Reply:
Done